shell/components/cards/AppCard.vue

```diff
@@ -0,0 +1,499 @@
+<script setup>
```

Comment:
Think the idea was for this to be a `rancher-component`.


In terms of name we were going to go with three generic cards that serve three purposes. Check with @oboc-sts about those purposes and come up with a nice name for this one. Feel free to discuss here or in slack if it helps.